be-python-flask/files/docker/Dockerfile

```diff
@@ -1,71 +1,24 @@
-FROM python:3.6.10-slim
+FROM python:3.8-buster
```

Comment:
Should this also be switched to use the rh-python packages so it is matching the agent?

Reply:
In this case I would prefer (also since other users using the quickstarter noted this) the official python provided ones. On the ds python related ones we are already providing the centos7/rhel7 though. 
At the end one can switch the FROM to whatever preference/requirement.
On the other side, I do not see any issue on providing different FROMs in different places if at least all of them provide the python 3.8 runtime and dependencies.
So, in this case I would not switch.